test/nightly/flink_validation_fhir_high_availability_spec.rb

```diff
@@ -5,18 +5,18 @@
 require_relative '../env'
 
 # Prior to running this test:
-#   - run `./gradlew build publishToMavenLocal` in the project's root directory to build the hri-flink-validation-fhir jar
+#   - run `./gradlew build publishToMavenLocal` in the project's root directory to build the flink-validation-fhir jar
```

Comment:
I think the changes to this file are in reverse.  Like David mentioned you change `branch_name` to `travis_branch` and here you're removing the `hri-` prefix.

Reply:
Fixed.